presto-elasticsearch/src/test/java/com/facebook/presto/elasticsearch/TestElasticsearchIntegrationSmokeTest.java

```diff
@@ -90,4 +95,35 @@ public void testDescribeTable()
                 .row("comment", "varchar", "", "").build();
         assertEquals(actualResult, expectedColumns, format("%s != %s", actualResult, expectedColumns));
     }
+
+    @Test
+    public void testCaseSensitiveField()
+    {
+        String indexName = "machine";
+
+        Map<String, Object> docAsSource = new HashMap<>();
+        docAsSource.put("ProductionCount", 32L);
+        docAsSource.put("Name", "A Big Machine");
+
+        addElasticDocumentToIndex(indexName, docAsSource);
+        embeddedElasticsearchNode.getClient().admin().indices().refresh(refreshRequest(indexName)).actionGet();
+
+        MaterializedResult actualColumns = computeActual("SELECT name, ProductionCount FROM telemetry.Machine").toTestTypes();
```

Comment:
define a string for "telemetry.Machine"
BTW, I think we could have a better table name and field name

Reply:
I'm all for a better names, but I had no idea.  Do you have any suggestion?